app/scss/imports/modal.scss

```diff
@@ -61,3 +61,16 @@
   }
 }
 
+.modal-close-cross {
+  position: relative;
+  left: 94%;
+  border: none;
+  background-color: transparent;
+  opacity: .5;
+  &:hover {
+    opacity: 1;
```

Comment:
maybe add a little transition so it fades in and out? not totally necessary but i'd like it if you had the time